packages/vscode-extension/src/webview/providers/ProjectProvider.tsx

```diff
@@ -26,6 +26,7 @@ const ProjectContext = createContext<ProjectContextProps>({
       longitude: 19.965474,
       isDisabled: false,
     },
+    locale: "en_US",
```

Comment:
why is this change here?

Reply:
It was missed before and ts was not happy, it does not effect the webview much at all as it never uses default context.